src/measure_theory/measurable_space_def.lean

```diff
@@ -302,7 +302,7 @@ def generate_from (s : set (set α)) : measurable_space α :=
   measurable_set_Union := generate_measurable.union }
 
 lemma measurable_set_generate_from {s : set (set α)} {t : set α} (ht : t ∈ s) :
-  (generate_from s).measurable_set' t :=
+  @measurable_set _ (generate_from s) t :=
```

Comment:
Is this what the following is syntax for?
```suggestion
  measurable_set[generate_from s] t :=
```

Reply:
I can't use this syntax in that file since its localized in another section of that file